powershell/resources/templates/longRunningOperationMethod.ejs

```diff
@@ -33,6 +33,6 @@
         public async System.Threading.Tasks.Task<<%-method.language.default.responseType%>> <%-method.language.default.name%>WithHttpMessagesAsync(<%-method.language.default.asyncMethodParameterDeclarationWithCustomHeader%>)
         {
                 // Send Request
-                <%-method.language.default.responseType%> _response = await Begin<%-method.language.default.name%>WithHttpMessagesAsync(<%-method.language.default.asyncMethodInvocationArgs%>).ConfigureAwait(false);
+                <%-method.language.default.responseType%> _response = await Begin<%-method.language.default.name%>WithHttpMessagesAsync(<%-method.language.default.asyncMethodInvocationArgs.replace('null', 'customHeaders')%>).ConfigureAwait(false);
```

Comment:
Do you confirm whether there is only one 'null' in asyncMethodInvocationArgs?

Reply:
yes, from previous commit:
https://github.com/dolauli/autorest.powershell/blob/457b43bbc4f52f89682a2b70e2fef0803e5574f5/powershell/plugins/sdk-tweak-model.ts#L148